CHANGELOG.md

```diff
@@ -16,6 +17,9 @@
         * ``identchars`` is now ignored. The standardlibrary cmd uses those characters to split the first "word" of the input, but cmd2 hasn't used those for a while, and the new parsing logic parses on whitespace, which has the added benefit of full unicode support, unlike cmd or prior versions of cmd2.
         * ``set_posix_shlex`` function and ``POSIX_SHLEX`` variable have been removed. Parsing behavior is now always the more forgiving ``posix=false``.
         * ``set_strip_quotes`` function and ``STRIP_QUOTES_FOR_NON_POSIX`` have been removed. Quotes are stripped from arguments when presented as a list (a la ``sys.argv``), and present when arguments are presented as a string (like the string passed to do_*).
+    * Enhanced the ``py`` console in the following ways
+        * Added tab completion of Python identifiers instead of **cmd2** commands
```

Comment:
So you made the built in default Python shell have tab completion similar to IPython?  Cool!